lib/menu_module.rb

```diff
@@ -0,0 +1,39 @@
+module Menu
+
+  attr_reader :view_menu
+
+  STARTERS = {:soup => 1.50,
+  :salad => 1.00,
+  :scallops => 2.50}.freeze
+  MAINS = {:steak => 7.50,
+  :lasagne => 5.50,
+  :lobster => 8.00 }.freeze
+  DESSERTS = {:tiramisu => 2.50,
+  :cheesecake => 2.00,
+  :profiteroles => 1.50}.freeze
+  SIDES ={:bread => 0.50,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)